windowsBase/windowsBasePull.ps1

```diff
@@ -1,3 +1,12 @@
 $ErrorActionPreference = "Stop"
 
-Write-Output "TODO: Complete logic for pulling windows docker images"
+Function pull_images() {
+  echo "RES_IMG_VER_NAME=$RES_IMG_VER_NAME"
+
+  foreach ($IMAGE_NAME in Get-Content "images.txt") {
```

Comment:
I guess images.txt won't be present in the AMI machine, so we need to either transfer it or introduce `IMAGE_NAMES_SPACED` like we do for ubuntu and use it here.